open-api/rest-catalog-open-api.yaml

```diff
@@ -1630,6 +1758,76 @@ components:
         - $ref: '#/components/schemas/OAuthClientCredentialsRequest'
         - $ref: '#/components/schemas/OAuthTokenExchangeRequest'
 
+    ScanReportRequest:
+      type: object
+      required:
+        - scan-report
+      properties:
+        scan-report:
+          $ref: '#/components/schemas/ScanReport'
+
+    ScanReport:
+      type: object
+      required:
+        - table-name
+        - snapshot-id
+        - filter
+        - projection
+        - metrics
+      properties:
+        table-name:
```

Comment:
Do we need to send the table name if the report is sent to the table's metrics resource?

Reply:
I think we still might want to send it, independently from whether the table name is part of the REST URL

Reply:
When would we use it and not ignore it?